src/appellate/appellate.js

```diff
@@ -369,6 +369,11 @@ AppellateDelegate.prototype.onDocumentViewSubmit = function (event) {
 
   let title = document.querySelectorAll('strong')[1].innerHTML;
   let dataFromTitle = APPELLATE.parseReceiptPageTitle(title);
+
+  if (dataFromTitle.att_number == 0 && this.queryParameters.get('attNum')){
```

Comment:
Hm, a thought. If we're adding query parameters just for our own sake, should we namespace them? In other words, could we make any parameter we use start with recap, say? `recapAttNum`?

Reply:
We can namespace the parameters. That's a great idea, I will change The set() method of the [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface.